app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,46 @@
+class PostsController < ApplicationController
+
+  
+ 
+  def show
```

Comment:
would recommend using only 1 space to separate isolated chunks of code, max 2. never 3.